src/components/30-organisms/footer/index.js

```diff
@@ -0,0 +1,279 @@
+// TODO fix that stuff
+/* eslint-disable import/no-extraneous-dependencies */
+import { CaretSvg } from '@axa-ch/materials/icons';
+import '@axa-ch/container';
+import { LitElement, html, svg, css, unsafeCSS } from 'lit-element';
+import defineOnce from '../../../utils/define-once';
+import styles from './index.scss';
+
+class AXAFooter extends LitElement {
+  static get tagName() {
+    return 'axa-footer';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      dynamic: { type: Boolean },
```

Comment:
What does dynamic mean? 1 line of comment would help a lot for the rest of the file...

Reply:
Hm, it is explained in the readme. We use this terminology also in other places. But I agree, it's not that readable. But before we start writing comments everywhere, I would vote, that we find another name that is more readable.

Reply:
What with: preventdefault ? which is what natively happens

Reply:
Writing comments is good, I requested 1 comment, not "everywhere". Just do it :-)

Reply:
@LucaMele I don't like "preventDefault" in this context. Using "native terminology" to express functionality dedicated to your component does not sound good to me. Example: Someone reads this and thinks "hmmm preventdefault, which event gets prevented?" It opens unnecessary other questions. In the end, in this case, ONLY clickevents get prevented. Not all events.

@markus-walther The fact, that you noticed it and were confused tells me, that we should find something better. Having an ugly property name and use comments and readmes to explain it is IMHO bad, if we could agree on a better name. How about a boolean with the name `clickevent`?

Reply:
Attribute is now renamed and commented.